packages/react-components/src/hooks/useAlarms/useAlarms.ts

```diff
@@ -57,10 +64,11 @@ function useAlarms<T>(options?: UseAlarmsOptions<T>): (T | AlarmData)[] {
    * Fetch latest asset property values for alarms with a state property.
    * Data should be available for all alarms fetched for an asset.
    */
-  const statePropertyAlarmData = useLatestAlarmPropertyValue({
+  const statePropertyAlarmData = useAlarmState({
```

Comment:
If we still wanna do these cascading hooks we need to move this to the end. Since not all alarms fetched here are alarms we want to return, they are filtered based on the inputProperty request.

Reply:
Yes that's a really good point. Let's address that in a separate change though? I know we are going to have to modify how we are managing the alarm data state too. we can likely combine those two updates.